test/unit/transloadit/test_api.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+describe Transloadit::ApiModel do
+  before do
+    @transloadit = Transloadit.new(:key => '')
+  end
+
+  it 'must allow initialization' do
+    Transloadit::ApiModel.new(@transloadit).
+      must_be_kind_of Transloadit::ApiModel
+
+    Transloadit::Template.new(@transloadit).
+      must_be_kind_of Transloadit::Template
+  end
+
+  describe 'when initialized' do
+    before do
+      @foo = 'foo'
+      @bar = 'bar'
+
+      @api = Transloadit::ApiModel.new @transloadit,
+        :foo => @foo,
+        :bar => @bar
+    end
```

Comment:
@rmehner yep! I agree, I previously didn't know about it but reading [this](http://ruby-journal.com/minitest-let-is-lazy/) makes it clear. This way all variables can be declared in the same place without performance implications